projects/ccd-case-ui-toolkit/src/lib/shared/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { SessionStorageService } from "./services";
+
+export const USER_DETAILS = 'userDetails';
+export const PUI_CASE_MANAGER = 'pui-case-manager';
+export const JUDGE = 'judge';
+
+export function isCaseworker(sessionStorageService: SessionStorageService): boolean {
```

Comment:
This doesn't quite do what it says on the tin. It might be better to call it isInternalUser, because it returns true for judges too